arcgis-rest-api/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for arcgis-to-geojson-utils 10.4
+// Project: http://resources.arcgis.com/en/help/arcgis-rest-api/
+// Definitions by: Jeff Jacobson <https://github.com/JeffJacobson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace ArcGis.Rest {
```

Comment:
That shouldn't be a problem. Would that just involve changing the keyword `namespace` to `module`, or are there other changes that need to be made?